test/Batch.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Thanks for splitting these tests out into a new file, good choice 👍 

Minor details: `'use strict';` is no longer needed, as I recall it's a relic from the Node.js 6 and earlier days. Goes to show what Peter said about this code not having been touched much in a while.